<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-09-28T10:33:24 IST" name="Default suite" finished-at="2024-09-28T10:34:05 IST" duration-ms="41104">
    <groups>
    </groups>
    <test started-at="2024-09-28T10:33:24 IST" name="Default test" finished-at="2024-09-28T10:34:05 IST" duration-ms="41104">
      <class name="SpecificationConcept.ResponseSpecBuilderTest">
        <test-method signature="get_res_Spec_200_OK_Test()[pri:0, instance:SpecificationConcept.ResponseSpecBuilderTest@6340e5f0]" started-at="2024-09-28T10:33:24 IST" name="get_res_Spec_200_OK_Test" finished-at="2024-09-28T10:34:05 IST" duration-ms="41070" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[4 expectations failed.
Expected status code <200> but was <522>.

Expected content-type "JSON" doesn't match actual content-type "text/html; charset=UTF-8".

XML path $.size() doesn't match.
Expected: <5>
  Actual: <1>

XML path id doesn't match.
Expected: a collection with size <5>
  Actual: <gorest.co.in | 522: Connection timed outnullnullnullnullnullnullConnection timed outError code 522cloudflare.com2024-09-28 05:04:05 UTCnullnullYou
    
    Browser
    
  WorkingnullnullMarseille
    Cloudflare
    Workingnullnullgorest.co.in
    
    Host
    
  ErrorWhat happened?The initial connection between Cloudflare's network and the origin web server timed out. As a result, the web page can not be displayed.What can I do?If you're a visitor of this website:Please try again in a few minutes.If you're the owner of this website:Contact your hosting provider letting them know your web server is not completing requests. An Error 522 means that the request was able to connect to your web server, but that the request didn't finish. The most likely cause is that something on your server is hogging resources.Additional troubleshooting information here.8ca12fe12f2170f4•Click to reveal110.226.182.106•Performance & security byCloudflare(function(){function d(){var b=a.getElementById("cf-footer-item-ip"),c=a.getElementById("cf-footer-ip-reveal");b&&"classList"in b&&(b.classList.remove("hidden"),c.addEventListener("click",function(){c.classList.add("hidden");a.getElementById("cf-footer-ip").classList.remove("hidden")}))}var a=document;document.addEventListener&&a.addEventListener("DOMContentLoaded",d)})();>
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 4 expectations failed.
Expected status code <200> but was <522>.

Expected content-type "JSON" doesn't match actual content-type "text/html; charset=UTF-8".

XML path $.size() doesn't match.
Expected: <5>
  Actual: <1>

XML path id doesn't match.
Expected: a collection with size <5>
  Actual: <gorest.co.in | 522: Connection timed outnullnullnullnullnullnullConnection timed outError code 522cloudflare.com2024-09-28 05:04:05 UTCnullnullYou
    
    Browser
    
  WorkingnullnullMarseille
    Cloudflare
    Workingnullnullgorest.co.in
    
    Host
    
  ErrorWhat happened?The initial connection between Cloudflare's network and the origin web server timed out. As a result, the web page can not be displayed.What can I do?If you're a visitor of this website:Please try again in a few minutes.If you're the owner of this website:Contact your hosting provider letting them know your web server is not completing requests. An Error 522 means that the request was able to connect to your web server, but that the request didn't finish. The most likely cause is that something on your server is hogging resources.Additional troubleshooting information here.8ca12fe12f2170f4•Click to reveal110.226.182.106•Performance & security byCloudflare(function(){function d(){var b=a.getElementById("cf-footer-item-ip"),c=a.getElementById("cf-footer-ip-reveal");b&&"classList"in b&&(b.classList.remove("hidden"),c.addEventListener("click",function(){c.classList.add("hidden");a.getElementById("cf-footer-ip").classList.remove("hidden")}))}var a=document;document.addEventListener&&a.addEventListener("DOMContentLoaded",d)})();>

at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:67)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:500)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:484)
at org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:73)
at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrapNoCoerce.callConstructor(ConstructorSite.java:108)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:57)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:263)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:512)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:45)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:139)
at io.restassured.internal.ResponseSpecificationImpl.validate(ResponseSpecificationImpl.groovy:101)
at io.restassured.internal.ValidatableResponseOptionsImpl.spec(ValidatableResponseOptionsImpl.java:288)
at SpecificationConcept.ResponseSpecBuilderTest.get_res_Spec_200_OK_Test(ResponseSpecBuilderTest.java:68)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- get_res_Spec_200_OK_Test -->
      </class> <!-- SpecificationConcept.ResponseSpecBuilderTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
